CONTRIBUTING.md

```diff
@@ -21,18 +21,158 @@ To get an overview of the project, read the [README](README.md). Here are some r
+
+For those who are new to computational documents like .qmd files but have experience with tools like Google Docs or MS Word, the visual editor in RStudio is the most user-friendly way to begin working with Quarto. In the visual editor, you can use the toolbar buttons to insert images, tables, cross-references, and other elements. 
+
 #### Adding images (in the right folder, code to do so)
 
+The visual editor also simplifies inserting images and customizing their display. You can paste an image directly from your clipboard into the visual editor, which will save a copy of the image in your project directory and link to it. Alternatively, you can use the Insert > Figure / Image menu to browse for the image you want to insert or paste its URL. This menu also allows you to resize the image, add captions, alternative text, and links.
+
+The visual editor offers many additional features that become evident as you gain more experience with it.
+
+Most importantly, while the visual editor displays your content with formatting, it saves everything in plain Markdown. You can easily switch between the visual and source editors to view and edit your content in either format.
+
 #### Adding links (within page, within site, external to site)\]
 
+In Quarto, you can add links using the following Markdown syntax:
+
+ 1. Inline Links:
+    [Link Text] ``(https://example.com)``
+
+ 2. Reference Links:
+    [Link Text][1]
+
+   `` [1]: https://example.com``
+    
+In Quarto, you can also include links within various contexts, such as code blocks or embedded within other content structures, depending on the complexity of your document. However, the basic Markdown link syntax will generally cover most use cases.
+ 
+ ![Example for adding links](images/adding_links_example.png){width=300}
+ 
 #### Adding videos / other content
+##### **Adding Videos**
+You can embed videos in Quarto documents using HTML, Markdown, or specialised Quarto syntax. Here are some methods:
+
+1. Using a shortcode:
+``` {.markdown shortcodes="false"} 
+{{< video url >}}
+``` 
+  - For example, here we embed a YouTube video:
+ 
+``` {.markdown shortcodes="false"} 
+{{< video https://www.youtube.com/embed/wo9vZccmqwc >}} 
+``` 
+{{< video https://www.youtube.com/embed/wo9vZccmqwc >}} 
+
+2. Embedding videos in other formats:
+  - In HTML formats the video will be embedded within the document.
+  - For other formats, a simple link to the video will be rendered.
+
+##### **Adding Other Content**
+ - **Audio Files**:
+ You can embed audio files using the HTML <audio> tag.
+ - **Interactive Content with HTML Widgets**:
+ Quarto supports various HTML widgets that can add interactive content to your document. 
+
+##### Tips 
+- **Relative Paths**:
+Use relative paths for local files to ensure your document remains portable. 
+
+In R, relative paths refer to file or directory paths specified in relation to the current working directory. Relative paths help manage current working files.
```

Comment:
Can you add an extra line here along the lines of "If you're struggling with running your files across different paths in your working directories, then the `here` package can sometimes help you solve those types of issues."

The link for the here package is https://here.r-lib.org/ if you can add that as a link in the text as well.